testsuite/MDAnalysisTests/lib/test_distances.py

```diff
@@ -171,14 +171,20 @@ def test_capped_distance_return(npoints, box, query, method, min_cutoff):
 @pytest.mark.parametrize('box', boxes_1)
 @pytest.mark.parametrize('method', method_1)
 @pytest.mark.parametrize('min_cutoff', min_cutoff_1)
-def test_self_capped_distance(npoints, box, method, min_cutoff):
+@pytest.mark.parametrize('ret_dist', (False, True))
+def test_self_capped_distance(npoints, box, method, min_cutoff, ret_dist):
```

Comment:
@NinadBhat I tracked down the reason for the partial hits for you. Hopefully CodeCov is happy now... :crossed_fingers:

Reply:
100% diff coverage. Woohooo! :tada: